src/mocks/handlers/document-type.handlers.ts

```diff
@@ -9,9 +9,12 @@ export const handlers = [
 		const data = await req.json();
 		if (!data) return;
 
-		umbDocumentTypeData.insert(data);
+		// TODO: This is something that is missing in the Full model, but which we need to for the tree model. This should be fixed in the Full model.
+		data.parentId ??= null;
 
-		return res(ctx.status(200));
+		const created = umbDocumentTypeData.insert(data);
+
+		return res(ctx.status(200), ctx.set({'location': '/header/location/id/'+created.id}));
```

Comment:
Should the header value be such a weird URL-ish string? I believe the backend controller will send back the URL of the Management API that you need to call to get the new item?

Reply:
I've written a comment above.